pkg/apiOracle/valueOracle.go

```diff
@@ -100,11 +106,15 @@ func EnsureValueOracle() error {
 		return nil
 	}
 
-	cfg := config.GetConfig()
+	filterLog, err := logging.ApplyFilter(*cfg, ComponentName, logger)
+	if err != nil {
+		return errors.Wrap(err, "creating filter logger")
+	}
+	logWith := log.With(filterLog, "component", "valueOracle")
```

Comment:
no need to create new variables. Here and few other places
```suggestion
	logger, err := logging.ApplyFilter(*cfg, ComponentName, logger)
	if err != nil {
		return errors.Wrap(err, "creating filter logger")
	}
	logger := log.With(filterLog, "component", "valueOracle")
```